Cargo.toml

```diff
@@ -56,3 +56,12 @@ default-features = false
 
 [profile.release]
 debug = 2
+
+[profile.opt-size]
+inherits = "release"
+opt-level = "z"
+lto = "on"
+codegen-units = 1
+panic = "abort"
```

Comment:
This flag removes the backtraces in case of error ([docs](https://doc.rust-lang.org/cargo/reference/profiles.html#panic)).

(possibly a controversial flag, so I wanted to point it out)

Reply:
I don't think that's true? It only means that Drop impls won't run, you should still be able to get a backtrace (presuming there's sufficient debug info...)

Reply:
ok. sorry for my misunderstarding